experimenter/experimenter/experiments/models.py

```diff
@@ -837,6 +837,28 @@ def excluded_live_deliveries(self):
             )
         return matching
 
+    @property
+    def live_experiments_in_namespace(self):
```

Comment:
That's a fair point. My reasoning was that it is technically a calculated field (like start date or can edit). I think I will keep it this time since the others have already landed as properties, and we can remove all three if we want to make them methods

Reply:
+1 for consistency